MANUAL.md

```diff
@@ -146,6 +146,7 @@ WebDAV or S3, that work out of the box.)
 - StackPath
 - SugarSync
 - Tardigrade
+- Tencent Cloud Object Storage (COS)
```

Comment:
This is an auto generated file - can you drop this from the commit?

Reply:
Fixed.